generic_message.go

```diff
@@ -0,0 +1,55 @@
+package analytics
+
+var _ Message = GenericMessage(nil)
+
+// This type represents any event type sent in a track call as described in
```

Comment:
This comment seems inaccurate, it seems that the values may be more than track calls only.

Reply:
Ah, the intention there was to express any event send through the Tracking API, but I completely fumbled that description!